libres/lib/enkf/enkf_main_manage_fs.cpp

```diff
@@ -285,19 +250,18 @@ static bool enkf_main_case_is_initialized__(const enkf_main_type *enkf_main,
                                             bool_vector_type *__mask) {
     ensemble_config_type *ensemble_config =
         enkf_main_get_ensemble_config(enkf_main);
-    stringlist_type *parameter_keys =
-        ensemble_config_alloc_keylist_from_var_type(ensemble_config, PARAMETER);
+    std::vector<std::string> parameter_keys =
+        ensemble_config_keylist_from_var_type(ensemble_config, PARAMETER);
     bool_vector_type *mask;
     bool initialized = true;
     int ikey = 0;
     if (__mask != NULL)
         mask = __mask;
     else
         mask = bool_vector_alloc(0, true);
-
-    while ((ikey < stringlist_get_size(parameter_keys)) && (initialized)) {
+    while ((ikey < parameter_keys.size()) && (initialized)) {
```

Comment:
Suggestion but maybe different PR.
```cpp
    bool initialized =  enkf_config_node_has_node(config_node, fs, {.report_step = 0, .iens = iens});
    for(int ikey = 0; (ikey < parameter_keys.size()) && (initialized); ikey++) {
        const enkf_config_node_type *config_node = ensemble_config_get_node(
            ensemble_config, parameter_keys[ikey].c_str());
        for(int iens = 0; (iens < enkf_main->ens_size) && (initialized);iens++){
            if (bool_vector_safe_iget(mask, iens)) {
                initialized =
                    enkf_config_node_has_node(config_node, fs, {.report_step = 0, .iens = iens});
            }
        }
    }
```

Reply:
Yes, that reads better, I am planning a follow up PR to this where I clean up a bit, but wanted to separate it out to avoid scope creep and keep the diff small-ish if that is ok with you?